package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "reshuffle-connectorname-connector",
+  "name": "@reshuffle/wix-connector",
   "version": "0.0.1",
-  "description": "A Reshuffle _CONNECTOR_NAME_ connector",
+  "description": "A Reshuffle WIX database connector",
```

Comment:
I usually remove the 'A' as it is the official Reshuffle connector for WIX database